### PR TITLE
CASMCMS-7618 Update image refs in chart cray-ims

### PR DESCRIPTION
**Pull request by ecozzi-hpe**
## Summary and Scope

Update chart.yaml, et al,  for CASM-2670 changes.

## Issues and Related PRs

_List and characterize relationship to Jira/Github issues and other pull requests. Be sure to list dependencies._

* Resolves [CASMCMS-7618](https://connect.us.cray.com/jira/browse/CASMCMS-7618)

## Testing

_List the environments in which these changes were tested._

### Tested on:

  * mug

### Test description:

_How were the changes tested and success verified? If schema changes were part of this change, how were those handled in your upgrade/downgrade testing?_

- Were the install/upgrade-based validation checks/tests run (goss tests/install-validation doc)? No.
- Were continuous integration tests run? If not, why? No
- Was upgrade tested? Yes
- Was downgrade tested? Yes
- Were new tests (or test issues/Jiras) created for this change? No

## Risks and Mitigations

_Are there known issues with these changes? Any other special considerations?_
No

## Pull Request Checklist

- [x] Version number(s) incremented, if applicable
- [x] Copyrights updated
- [x] License file intact
- [x] Target branch correct
- [x] CHANGELOG.md updated
- [x] Testing is appropriate and complete, if applicable
- [x] [HPC Product Announcement](https://cray.slack.com/archives/C026TVCSXLH) prepared, if applicable

### CODE REVIEW

**Review by jsl-hpe — APPROVED** [review]

File: kubernetes/cray-ims/templates/post_upgrade_hook.yaml [hunk 1]
```diff
@@ -13,7 +13,7 @@ spec:
       restartPolicy: Never
       containers:
       - name: ims-post-upgrade-hook1-container
-        image: "baseos/alpine:3.13"
+        image: "arti.dev.cray.com/baseos-docker-master-local/alpine:3.13"
```
Comment (rbak-hpe) [inline kubernetes/cray-ims/templates/post_upgrade_hook.yaml:16]: Is this the correct image path?  Zach said arti.dev.cray.com would be deprecated in 1.2: https://cray.slack.com/archives/C02H5FEPHBK/p1636501974195300?thread_ts=1636501736.195200&cid=C02H5FEPHBK

Comment (ecozzi-hpe) [reply kubernetes/cray-ims/templates/post_upgrade_hook.yaml]: Updated to use artifactory.algol60.net/docker.io/alpine:latest

File: kubernetes/cray-ims/templates/post_upgrade_hook.yaml [hunk 2]
```diff
@@ -13,7 +13,7 @@ spec:
       restartPolicy: Never
       containers:
       - name: ims-post-upgrade-hook1-container
-        image: "baseos/alpine:3.13"
+        image: "artifactory.algol60.net/docker.io/alpine:latest"
```
Comment (rbak-hpe) [inline kubernetes/cray-ims/templates/post_upgrade_hook.yaml:16]: Does this need to be included in the chart annotations?

Comment (ecozzi-hpe) [reply kubernetes/cray-ims/templates/post_upgrade_hook.yaml:16]: Good catch. I'll ask Zach. But, seem like it should.

**Review by rbak-hpe — APPROVED** [review]

**Review by rkleinman-hpe — APPROVED** [review]

**Pull request merged**